templates/zerver/find_account.html

```diff
@@ -28,7 +28,10 @@ <h1 class="get-started">{{ _("Find your Zulip accounts") }}</h1>
                     <li>{{ email }}</li>
                     {% endfor %}
                 </ul>
-
+                {% trans %}
+                If you don't receive an email, you can
+                <a href="{{ current_url }}">find accounts for another email address</a>.
+                {% endtrans %}
```

Comment:
Let's have blank lines around this block -- it's easier to read that way.

Reply:
Made the necessary changes